docs/docs/04-standard-library/02-std/api-reference.md

```diff
@@ -2051,6 +2051,37 @@ The length of the string.
 ---
 
 
+### Struct <a name="Struct" id="@winglang/sdk.std.Struct"></a>
+
+Base class for all structs.
```

Comment:
I think we want to avoid confusing people by saying that structs are classes. Maybe we could tweak the description to something like
```suggestion
### Struct <a name="Struct" id="@winglang/sdk.std.Struct"></a>

Shared behavior for all structs.
```